ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/Testnet/Simulation/Node.hs

```diff
@@ -218,24 +223,26 @@ data NodeArgs =
 
 instance Show NodeArgs where
     show NodeArgs { naSeed, naDiffusionMode, naMbTime, naBootstrapPeers, naPublicRoots,
-                   naAddr, naPeerSharing, naLocalRootPeers, naLocalSelectionTargets,
+                   naAddr, naPeerSharing, naLocalRootPeers, naPeerTargets,
                    naDNSTimeoutScript, naDNSLookupDelayScript, naChainSyncExitOnBlockNo,
-                   naChainSyncEarlyExit, naFetchModeScript } =
-      unwords [ "NodeArgs"
+                   naChainSyncEarlyExit, naFetchModeScript, naConsensusMode } =
+      intercalate "\n  " [ "NodeArgs"
```

Comment:
Please don't do this as it makes it harder to copy paste the output to a unit test

Reply:
Done - some of these comments I've missed as they're not on the main conversation page, or they're folded in there by the algorithm with some other comments that I've addressed which is not convenient.